config/prow/jenkins/deploy.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
These steps in this script can be placed at the end of https://github.ibm.com/powercloud-secrets/secrets/blob/master/projects/k8s-prow/deploy.sh.
May be included as part of PR- https://github.ibm.com/powercloud-secrets/secrets/pull/25